src/ofxOssia.cpp

```diff
@@ -24,6 +31,31 @@ void ofxOssia::setup(const std::string& type,
     _root_node.setup (_device.getRootNode(), "ossia");
 }
 
+void explore(const ossia::net::node_base& node)
+{
+  for (const ossia::net::node_base* child : node.children_copy())
+  {
+    if (auto addr = child->getAddress())
+    {
+	std::cerr << ossia::net::osc_address_string(*addr) << " : "
```

Comment:
please don't leave too many debug messages, and if you do, instead use ossia::logger().info(), error(), warn(), etc. depending on the severity. It uses the [spdlog](https://github.com/gabime/spdlog#usage-example) library.